zeppelin-integration/src/test/java/org/apache/zeppelin/integration/InterpreterModeActionsIT.java

```diff
@@ -184,8 +184,7 @@ private void setPythonParagraph(int num, String text) {
   void testGloballyAction() throws Exception {
     try {
       //step 1: (admin) login, set 'globally in shared' mode of python interpreter, logout
-      InterpreterModeActionsIT interpreterModeActionsIT = new InterpreterModeActionsIT();
-      interpreterModeActionsIT.authenticationUser("admin", "password1");
+      authenticationUser("admin", "password1");
```

Comment:
This class also contains the method `authenticationUser`, we should only have the function once. My suggestion: not static in `AbstractZeppelinIT`.

Reply:
adopted